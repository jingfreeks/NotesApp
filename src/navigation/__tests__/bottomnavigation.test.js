import React from 'react';
import { renderWithProviders } from "@/utils/testframework";
import BottomTabScreen from '../bottomtabnavigation';


describe('Bottom navigation Screen', () => {
  it('Should work as expected to get snapshot', () => {
    const all = renderWithProviders(<BottomTabScreen />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
