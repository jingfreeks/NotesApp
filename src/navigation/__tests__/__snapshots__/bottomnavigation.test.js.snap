// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottom navigation Screen Should work as expected to get snapshot 1`] = `null`;
